css.editor/src/org/netbeans/modules/css/resources/CascadeStyleSheet.css.template

```diff
@@ -1,3 +1,24 @@
+<#--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+-->
```

Comment:
Please undo this change, then, exclude this file form rat. This file is user visible. (Tools > Templates > Other > Cascading Style Sheet : Open in editor)

![nb54-css editor-template](https://user-images.githubusercontent.com/738383/31821990-f8cf14ac-b5e1-11e7-8446-b0ccd098bd54.png)

Reply:
Please address the comment from @junichi11